sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/constraints/BaseConstraint.java

```diff
@@ -63,12 +63,13 @@ public boolean rely() {
 
   @Override
   public String toDDL() {
+    // The validation status is not included in the DDL output as it's not part of
+    // the Spark SQL syntax for constraints.
     return String.format(
-        "CONSTRAINT %s %s %s %s %s",
+        "CONSTRAINT %s %s %s %s",
         name,
         definition(),
         enforced ? "ENFORCED" : "NOT ENFORCED",
-        validationStatus,
```

Comment:
cc @aokolnychyi

Reply:
No. We have to remove it since the validation status is not supported in the parser.
We can still display it in the "DESC TABLE" command